final_task/rss_reader/rss_reader.py

```diff
@@ -0,0 +1,106 @@
+import logging
+
+from ConsoleArgument import get_console_argument
+from Handler import Handler
+from Handler import parse_to_json
+from ConsoleOut import print_array_of_news
+from ConsoleOut import print_array_of_dict
+from ConsoleOut import print_json
+from RssException import RssException
+from WorkWithCache import read_from_file
+from ConvertToHtmlAndPdf import create_html_news
+from ConvertToHtmlAndPdf import create_pdf_news
+from ConvertToHtmlAndPdf import convert_Dict_to_News
```

Comment:
If you are importing more than two objects from a module it is better to import the whole module itself.